src/Backend/Modules/ContentBlocks/Actions/ContentBlockDelete.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Backend\Modules\ContentBlocks\Actions;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Backend\Core\Engine\Base\ActionDelete as BackendBaseActionDelete;
+use Backend\Core\Engine\Model as BackendModel;
+use Backend\Core\Language\Locale;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Command\DeleteContentBlock;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlock;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Event\ContentBlockDeleted;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Exception\ContentBlockNotFound;
+
+/**
+ * This is the delete-action, it will delete an item.
+ */
+class ContentBlockDelete extends BackendBaseActionDelete
+{
+    public function execute(): void
```

Comment:
if https://github.com/forkcms/forkcms/pull/2090 gets merged before this one the logic here needs to be updated since it won't get merged because you changed everything in one big commit and thereby are breaking the git history